### DISCUSSION TITLE
StartOSS

### DISCUSSION THREAD

**Discussion by adamc199** [TriliumNext/Trilium#4458] [General]
I have been running trilium on an umbrel since it was first available but am switching to a start9 server running startOS. 

Not a developer, would be happy to return value to anyone who can help make this a service in StartOS

**Comment by Nriver** [comment]
You can simply use docker. https://github.com/zadam/trilium/wiki/Docker-server-installation

**Reply by adamc199** [reply]
I understand that startOS uses docker containers, but the process for me as a non developer creating a StaftOS service out if a docker container is pretty daunting. Hence my offer to return value in exchange for this work.
